itests/docker/Dockerfile

```diff
@@ -1,11 +1,12 @@
 ARG MARATHONVERSION=v1.6.322
 FROM mesosphere/marathon:$MARATHONVERSION
```

Comment:
too bad we cant just use the upstream image directly?

Reply:
I tried using it directly but your functional tests go deeper and we need `mesos-master` and `mesos-agent` for all tests to work.

BTW, have these tests is great, it would go as far as to say these are end-to-end <3